apps/processing/src/services/processing.service.ts

```diff
@@ -65,7 +73,12 @@ export class ProcessingService {
         });
 
         try {
-            await this.orchestrator.run(abortController.signal);
+            for (const orchestrator of this.orchestrators.values()) {
+                this.logger.info(`Starting orchestrator for chain ${orchestrator.chainId}...`);
+                orchestratorProcesses.push(orchestrator.run(abortController.signal));
+            }
+
+            await Promise.allSettled(orchestratorProcesses);
```

Comment:
Lets keep in mind that we should make all the orchestrators to be infinite running processes, never throw errors, and have automatic recovery. This is to avoid affecting the others. In case we have inifinite running processes and never throw errors, then Promise.allSettled or Promise.all both should be the same since none of the processes resolve. Apart from automatic recovery we should notify once there is a critical error that kills the specific orchestrator.

Reply:
agree that all and allSettled are the same here. You're suggesting to use all so we can catch early a possible Fatal error that killed the Orchestrator?

Reply:
Should be the same using `allSettled` or `all` because all are infinite processes and each of them should avoid bubbling up errors